selfdrive/ui/translations/main_pl.ts

```diff
@@ -0,0 +1,1336 @@
+        <source>right</source>
+        <translation>w prawo</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="211"/>
+        <source>Are you sure you want to reboot?</source>
+        <translation>Czy na pewno chcesz uruchomić ponownie urządzenie?</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="218"/>
+        <source>Disengage to Reboot</source>
+        <translation>Aby uruchomić ponownie, odłącz sterowanie</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="224"/>
+        <source>Are you sure you want to power off?</source>
+        <translation>Czy na pewno chcesz wyłączyć urządzenie?</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="231"/>
+        <source>Disengage to Power Off</source>
+        <translation>Aby wyłączyć urządzenie, odłącz sterowanie</translation>
+    </message>
+</context>
+<context>
+    <name>DriveStats</name>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="37"/>
+        <source>Drives</source>
+        <translation>Przejazdy</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="39"/>
+        <source>Hours</source>
+        <translation>Godziny</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="44"/>
+        <source>ALL TIME</source>
+        <translation>CAŁKOWICIE</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="46"/>
+        <source>PAST WEEK</source>
+        <translation>OSTATNI TYDZIEŃ</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>KM</source>
+        <translation>KM</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>Miles</source>
+        <translation>Mile</translation>
+    </message>
+</context>
+<context>
+    <name>DriverViewScene</name>
+    <message>
+        <location filename="../qt/offroad/driverview.cc" line="55"/>
+        <source>camera starting</source>
+        <translation>uruchamianie kamery</translation>
+    </message>
+</context>
+<context>
+    <name>InputDialog</name>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="71"/>
+        <source>Cancel</source>
+        <translation>Anuluj</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/widgets/input.cc" line="168"/>
+        <source>Need at least %n character(s)!</source>
+        <translation type="unfinished">
+            <numerusform></numerusform>
+            <numerusform></numerusform>
+            <numerusform></numerusform>
```

Comment:
Do you mind adding the new plural forms?

Reply:
Sure! I have just updated the PR 👍

Reply:
Thanks!